sputnikdao-factory2/src/lib.rs

```diff
@@ -24,6 +24,7 @@ const DAO_CONTRACT_NO_DATA: &str = "no data";
 // Gas & Costs for blob storage
 const GAS_STORE_CONTRACT_LEFTOVER: Gas = Gas(20_000_000_000_000);
 const ON_REMOVE_CONTRACT_GAS: Gas = Gas(10_000_000_000_000);
+const CREATE_COST: Balance = 5_000_000_000_000_000_000_000_000;
```

Comment:
Haha Trevor, should I remind you that we use 0 N as we speak? There are currently no constraints on this on mainnet.

Worst case scenario is that the user provides 5 N and the factory pays for the rest. Other alternatives include imposing a higher cost (6 N) from the Astro UI.

I think 5 should be enough, it has been 0 for ages and it still is until we do the upgrade.

Reply:
How did you come up with the 5.3 N? 🐶

Reply:
We shouldnt leave something bad, because it was previously forgotten. Let's find a good solution moving forward

Reply:
> How did you come up with the 5.3 N? 🐶

5.3 is the storage cost for DAO v3 store blob

Reply:
On store blob, the user is forced to provide the necessary deposit to cover the storage. So 5.3 is not necessary for the creation of the DAO, it's a mandatory deposit that you need to provide when you store a blob.

Reply:
This is what you need to provide when you're creating a new DAO:

TGas: 300
Bond: 5 NEAR

Why do you think 5 NEAR is not enough to handle the creation? After creating the DAO the user can deposit more NEAR if they want to. But just for the creation, I think 5 is more than enough.

Reply:
Current DAO storage costs less than 5 Near which is why UIs standardized to 5 Near. (last i checked it was 4.7 or something). 
When we move to new DAO v3, 5N means the DAO effectively starts locked and cannot execute proposals if bond is less than 0.7 near. Therefore we effectively are hoping that UIs attach more than 5.3 Near to cover the new base-cost of DAO storage needs (Since the DAO code is the same size as store_blob.)

When i mentioned store blob, it was to help you see how i arrived at 5.3 N, which can be the exact same need for the base DAO code size.

Reply:
I propose a solution is needed:
1. Setup a configurable variable for creation cost, changed by owner only
2. calculate the cost of storage required for the "default code blob" and ensure the attached deposit is higher.

Reply:
> TGas: 300
> Bond: 5 NEAR

by Bond, you mean deposit right?

Reply:
Yes, I meant deposit. I would go for B in this case.